devito/operator.py

```diff
@@ -188,6 +192,36 @@ def __init__(self, expressions, **kwargs):
 
     # Compilation
 
+    def _add_implicit(self, expressions):
+        """
+        Create and add to the expression tree any associated implicit
+        expressions.
+        """
+        updated_expressions = []
```

Comment:
nitpicking (for code homogeneity): throughout the codebase, I use the word "processed", or sometimes "ret", to collect the items that will be returned by the function/method

Reply:
Hmm, not sure I like 'processed' here since that's not what's happening. But will have a think.